ui-backend/intrigue-ui-app/src/main/resources/features.xml

```diff
@@ -29,7 +29,6 @@
         <feature>resource-bundle-locator</feature>
         <feature>platform-usng4j</feature>
         <feature>platform-configuration</feature>
-        <feature>platform-email</feature>
```

Comment:
Do workspace subscriptions/email notifications work without this?

Reply:
They probably would, since this is installed on top of a DDF system which **must** have `security-core-services` installed. I'll go ahead and explicitly add `security-core-services` though.